components/inputs/docs/input-number.md

```diff
@@ -0,0 +1,52 @@
+# Number Inputs
+
+The `<d2l-input-number>` element is a wrapper around the `<d2l-input-text>` component. It's intended for inputting numbers only.
+
+![example screenshot of number input](../screenshots/number.gif?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/input-number.js';
+</script>
+<d2l-input-number
+  label="Label"
+  value="0">
+</d2l-input-number>
+```
+
+**Properties:**
+
+| Property | Type | Description |
+|--|--|--|
+| `label` | String, required | Label for the input. |
```

Comment:
These seem to be alphabetical except for `label`

Reply:
This is because it's a `required` attribute. I see that `label` is always first for other docs as well.

Reply:
Ah ok cool 👍